docs/samples/fluid/README.md

```diff
@@ -0,0 +1,343 @@
+# KServe with Fluid on LLM (Large Language Model) Acceleration
+
+In this example we will show how to use [Fluid](https://github.com/fluid-cloudnative/fluid) to improve autoscaling performance on LLM (Large Language Model) in KServe.
+
+## DISCLAIMER
+
+This Proof of Concept (PoC) is a collaborative effort between KServe and Fluid communities to integrate KServe with Fluid. The purpose of this PoC is to experiment with the use of Fluid to improve the autoscaling performance on LLMs.
+
+It is important to note that this PoC has not been tested in a production environment and is intended for experimental purposes only. Therefore, if you decide to try it out, please carefully validate it in your use case before deploying it in a production environment.
+## Context
+
+With the increasing popularity of large language models (LLMs), many companies are now deploying them via KServe. However, due to their typical size ranging from a few GBs to several hundred GBs, it can take a considerable amount of time for the KServe Storage Initializer to pull them from remote storage. As a result, the start-up time for an inference service can take anywhere from a few minutes to over 30 minutes due to the lengthy model downloading and loading times. This makes the serverless autoscaling function of KServe less effective for LLM use cases.
+To improve performance, it becomes crucial to cache the model artifacts rather than pulling them from remote storage during the autoscaling process. Since large language models are often served on GPU instances, which can be expensive, autoscaling becomes a helpful tool for reducing costs by allowing resources to be scaled up or down based on demand.
+
+## Prerequisite
+
+* [KServe](https://kserve.github.io/website/master/admin/serverless/serverless/)
+  * Please follow this [guideline](https://kserve.github.io/website/master/developer/developer/#deploy-kserve-from-master-branch) to deploy KServe from `master` branch
+  * Enable node selector in KServe InferenceService via KNative
+
+    ```sh
+
+    kubectl patch configmap/config-features \
+      --namespace knative-serving \
+      --type merge \
+      --patch '{"data":{"kubernetes.podspec-nodeselector":"enabled"}}'
+    ```
+  * Enable Direct VolumeMount for PVC In KServe
+
+    ```sh
+    # edit inferenceservice-config and update enableDirectPvcVolumeMount to true
+    kubectl edit configmap inferenceservice-config  -n kserve
+
+    #   storageInitializer: |-
+    #     {
+    #         "image" : "kserve/storage-initializer:latest",
+    #         "memoryRequest": "100Mi",
+    #         "memoryLimit": "1Gi",
+    #         "cpuRequest": "100m",
+    #         "cpuLimit": "1",
+    #         "storageSpecSecretName": "storage-config",
+    #         "enableDirectPvcVolumeMount": false        # change to true
+    #     }
+    ```
+
+* [Fluid](https://github.com/fluid-cloudnative/fluid/blob/master/docs/en/userguide/get_started.md)
+  * Deploy Fluid from `master` branch
+
```

Comment:
You can use the following way to install fluid latest version.

```bash
$ helm repo add fluid https://fluid-cloudnative.github.io/charts
$ helm repo update
$ helm search repo fluid --devel
NAME            CHART VERSION   APP VERSION     DESCRIPTION                               
fluid/fluid     0.9.0-alpha.7   0.9.0-ac1fb47   A Helm chart to deploy fluid in Kubernetes

$ helm install fluid fluid/fluid --devel
```

Reply:
Thanks @cheyang I will test it and update it accordingly.